src/menu.rs

```diff
@@ -35,7 +33,7 @@ pub struct MenuState {
 impl MenuState {
     pub fn new() -> MenuState {
         MenuState {
-            name: String::from(whoami::username()),
+            name: String::new(),
```

Comment:
I think we should default to `whoami`, that way you get a name even if you're too lazy to write it :)

Reply:
Yes, I agree.

Reply:
It is in client.rs now because setting it in MenuState::new wouldn't let you keep your name